client/packages/system/src/Stock/Components/InventoryAdjustment/AdjustmentStats.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  useTranslation,
+  Box,
+  Typography,
+  SimpleStatistic,
+  UNDEFINED_STRING_VALUE,
+} from '@openmsupply-client/common';
+import { StockLineRowFragment } from '../../api';
+
+export const ItemDetailAndStats = ({
+  stockLine,
+  variation,
+}: {
+  stockLine: StockLineRowFragment;
+  variation: number;
+}) => {
+  const t = useTranslation();
+
+  const {
+    item: { code, name },
+    packSize,
+    totalNumberOfPacks,
+    availableNumberOfPacks,
+  } = stockLine;
+  return (
+    <Box
+      sx={{
+        borderWidth: 4,
+        borderRadius: '16px',
+        borderStyle: 'solid',
+        borderColor: 'border',
+        padding: 1,
+        margin: '0 auto',
+        width: '520px',
+      }}
+    >
+      <Box sx={{ paddingX: 1, marginBottom: 1 }}>
+        <Typography>
+          {t('label.code')}:
+          <Typography component="span" color="gray.dark">
+            {` ${code} | `}
+          </Typography>
+          {t('label.pack-size')}:
+          <Typography component="span" color="gray.dark">
+            {` ${packSize}`}
+          </Typography>
+        </Typography>
+
+        <Typography sx={{ fontWeight: 500, fontSize: '22px' }}>
+          {name}
+        </Typography>
+      </Box>
+
+      <Box sx={{ display: 'flex', gap: 1 }}>
+        <AdjustmentStats
+          originalValue={availableNumberOfPacks}
+          variation={variation}
+          label={t('label.available-packs')}
+        />
+        <AdjustmentStats
+          originalValue={totalNumberOfPacks}
+          variation={variation}
+          label={t('label.packs-on-hand')}
+        />
+      </Box>
+    </Box>
+  );
+};
+
+export const AdjustmentStats = ({
+  originalValue,
+  variation,
+  label,
+}: {
+  originalValue: number;
+  variation: number;
+  label: string;
+}) => {
+  const t = useTranslation();
+
+  return (
+    <Box
+      sx={{
+        backgroundColor: 'background.secondary',
+        padding: '1em',
+        borderRadius: '16px',
+      }}
+    >
+      <Typography
+        sx={{ fontWeight: '600', fontSize: '0.875em', textAlign: 'center' }}
+      >
+        {label}
+      </Typography>
+      <Box
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'end',
+          gap: '1em',
+          marginY: '0.25em',
+        }}
+      >
+        <SimpleStatistic
+          label={t('label.current')}
+          value={originalValue}
+          color={'secondary.main'}
+        />
+        <Box
+          sx={{
+            width: '1px',
+            backgroundColor: 'secondary.main',
+            height: '-webkit-fill-available',
```

Comment:
Oh, it's this that's causing the weird stretching in Safari -- I don't think you need it all do you?

Reply:
Probably same for the `width` one earlier